modules/candidate_profile/test/candidate_profileTest.php

```diff
@@ -97,12 +97,10 @@ function testCandidateProfileWithoutProjectPermissions()
      */
     function testCandidateProfileInstrumentLink1()
     {
-        $this->safeGet($this->url . "/candidate_profile/900000/");
+        $this->safeGet($this->url . "/candidate_profile/300001/");
         $this->safeFindElement(
             WebDriverBy::cssSelector(
-                "#card2 > div:nth-child(1) >".
-                "div:nth-child(1)>div:nth-child(1)>div:nth-child(1)>div:nth-".
-                "child(1)> div:nth-child(2) > h4:nth-child(1) > a:nth-child(1)"
+                "#card2 > div > div > div > div:nth-child(1) > div > div:nth-child(2) > h4 > a"
```

Comment:
$this->safeClick(
            WebDriverBy::cssSelector(
                '#card2 > div > div > div > div:nth-child(1) > div > div:nth-child(2) > h4 > a'
            );
        );